code/DDSCodeTester.cpp

```diff
@@ -7684,6 +7685,58 @@ void pubsub_api_example_create_entities()
     //!--
 }
 
+void statistics_defined_interface()
+{
+    using namespace eprosima::fastdds::rtps;
+    using namespace eprosima::fastdds::dds;
+
+    // STATISTICS_DEDICATED_INTERFACE
+    // --- Device 1 ---
+
+    // Create DomainParticipant 1
+    DomainParticipantQos pqos1;
+    // Set NetmaskFIlter = ON to enable communication only between locators on the same subnetwork
+    pqos1.transport().netmask_filter = NetmaskFilterKind::ON;
+    pqos1.wire_protocol().ignore_non_matching_locators = true; // Required if not defining an allowlist or blocklist
```

Comment:
This doesn't work if `ignore_non_matching_locators` is also set in Datawriter/Datareader. At least I will write a note here.